src/levelset/levelSetSegmentationObject.cpp

```diff
@@ -1817,6 +1817,92 @@ LevelSetIntersectionStatus LevelSetSegmentationBaseObject::_isCellIntersected(lo
     return LevelSetObject::_isCellIntersected(id, distance, mode);
 }
 
+/*!
+ * Check if the specified interface intersects the zero-levelset iso-surface.
+ *
+ * The iso-surface is considered planar. Calculate the intersection
+ * as a segment defined by two points and the part of the interface belonging to the subspace
+ * of negative level-set. If the interface is not intersected, an empty polygon is returned. 
+ * The interface belonging to the possitive level-set subspace is returned if the
+ * "positivePart" argument is true.
+ *
+ * The intersection is computed by considering the zero-levelset as a curved surface.
+ *
+ * @param[in] id is the interface index
+ * @param[in] positivePart when true the part of the  interface occupying the positive levelset area is returned
```

Comment:
This seems an argument to decide which portion of the intersect interface should be returned. It that's the case, we could mimic what ParaView does: let's define a default behaviour and then pass an argument called "invert" which inverts which part of the intersected interfae is considered.